integration-tests/features/steps/authorization.py

```diff
@@ -87,3 +87,19 @@ def check_api_token(context):
         print(context.kerb_request)
         raise
     assert j["token"]
+
+
+@when("I acquire the user_key")
+def acquire_user_key(context):
```

Comment:
I'd rename it to acquite_user_key_for_3scale, dtto for the `@when clause` (all the clauses are joined into one group)